src/components/visual/side-panel/side-panel.tsx

```diff
@@ -10,10 +9,12 @@ import { SidePanelSideEnum } from './side-panel.types';
 })
 export class SidePanel {
   @Prop() isOpen: boolean = false;
-  @Prop() side: SidePanelSideEnum = SidePanelSideEnum.RIGHT;
   @Prop() panelClassName?: string;
+  @Prop() panelTitle: string;
+  @Prop() withBackButton?: boolean;
 
   @Event() close: EventEmitter;
+  @Event() back: EventEmitter;
```

Comment:
back? --> if provided, will have back button & no need for withBackButton

Reply:
For some reason, event emitters always seem to be existing. This is what gets returned when I don't pass any `onBack` value to the `<mvx-side-panel />` component and then I log the `back` event emitter in it. So we need to leverage on another property for this case.
<img width="256" alt="image" src="https://github.com/user-attachments/assets/63ce9ffb-c919-4c0f-892b-891055e690ef" />